doc/users/whats_new/ticks_rcparams.rst

```diff
@@ -3,4 +3,8 @@ New rcParams
 
 The parameters `xtick.top`, `xtick.bottom`, `ytick.left`
 and `ytick.right` were added to control where the ticks
-are drawn.
\ No newline at end of file
+are drawn.
+
+``hist.bins`` to control the default number of bins to use in
```

Comment:
There is inconsistency with notation for the rcparams. Here, double backticks are used. Above, single backticks are used.